include/libdnf/rpm/solv_sack.hpp

```diff
@@ -127,6 +127,8 @@ class SolvSack {
     /// Creates system repository and loads it into SolvSack. Only one system repository can be in SolvSack.
     void create_system_repo(bool build_cache = false);
 
+    void dump_debugdata(const std::string & dir);
```

Comment:
I am not sure whether we want to keep the functionality exactly in this place, because there will be an overlap with goal debug. Please could you add TODO that we have to reconsider it in future?

Reply:
I can add the TODO; I've noticed the dump is normally used with a transaction to dump its data as well, but I found it quite useful just to dump the repositories, when you may not have any transaction at all.